src/plugins/axios.js

```diff
@@ -1,7 +1,9 @@
 import axios from 'axios'
 
-const isDevelopment =
-  process.env.NODE_ENV === 'development'
+const isDevelopment = (
+  import.meta.env.MODE ===
```

Comment:
isDevelopment can be patched by setting your env to development. a more robust solution would be getting `app.isPackaged`.

Reply:
@xyloflake Ok, will check if it's working.